src/Utils/Hooks/useMedia.ts

```diff
@@ -0,0 +1,41 @@
+/**
+ * Thanks! https://github.com/olistic/react-use-media/
+ */
+
+import { useEffect, useState } from "react"
+
+/**
+ * Checks to see if the browser matches a particular media query
+ *
+ * @example
+
+    import { themeProps } from '@artsy/palette'
+    import { useMedia } from 'Utils/Hooks/useMedia'
+
+    const App = () => {
+      const isMobile = useMedia(themeProps.mediaQueries.sm)
+
+      return (
+        <div>Mobile view? {isMobile}</div>
+      )
+    }
+ */
+export function useMedia(
+  mediaQueryString: string,
+  { initialMatches = true } = {}
+) {
+  const [matches, setMatches] = useState(initialMatches)
+
+  useEffect(() => {
+    const mediaQueryList = window.matchMedia(mediaQueryString)
+    setMatches(mediaQueryList.matches)
+    const handleChange = event => setMatches(event.matches)
+
+    mediaQueryList.addEventListener("change", handleChange)
+    return () => {
+      mediaQueryList.removeEventListener("change", handleChange)
+    }
+  }, [mediaQueryString])
+
+  return matches
+}
```

Comment:
Grabbed this hook from https://github.com/olistic/react-use-media/. Is MIT. Didn't want to have to install a new package for something this simple and something we might change.